Src/coffeescript/cql-execution/src/example/CMS146v2_CQM.cql

```diff
@@ -46,7 +46,7 @@ valueset "Ambulatory/ED Visit": '2.16.840.1.113883.3.464.1003.101.12.1061'
 valueset "Antibiotic Medications": '2.16.840.1.113883.3.464.1003.196.12.1001'
 valueset "Group A Streptococcus Test": '2.16.840.1.113883.3.464.1003.198.12.1012'
 
-parameter MeasurementPeriod Interval<DateTime>
+parameter MeasurementPeriod default Interval[DateTime(2013, 1, 1, 0, 0, 0, 0), DateTime(2014, 1, 1, 0, 0, 0, 0))
```

Comment:
The framework was bombing on line 97 of https://github.com/cqframework/clinical_quality_language/blob/7a5b59d747edee49a266107934cfd129209e56e7/Src/coffeescript/cql-execution/src/elm/clinical.coffee

The problem appeared to be that no measurement period was actually being specified.  Putting a default interval in the CQL seemed to be a reasonable solution (although not the only one).